components/cards/Card.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import * as S from '@components/cards/Card.style';
+import Image from 'next/image';
+import Chip from '@components/chips/Chip';
+
+// 리소스
+import calendarIcon from '@public/icons/calendar.svg';
+import { TColorKey } from '@components/chips/Chip.type';
+
+type TagProps = {
+  text: string;
+  color: TColorKey;
+};
+
+export type CardProps = {
+  imageUrl: string | null;
+  title: string;
+  tags: TagProps[];
+  dueDate: string;
+  assignee: { nickname: string };
+};
+
+function Card(props: CardProps) {
+  return (
+    <S.CardContainer>
+      {props.imageUrl ? (
+        <S.CardImage>
+          <Image src={props.imageUrl} alt={'Card Image'} fill />
+        </S.CardImage>
+      ) : null}
```

Comment:
```suggestion
      {props.imageUrl && (
        <S.CardImage>
          <Image src={props.imageUrl} alt='Card Image' fill />
        </S.CardImage>
      )}
```
alt작성 시 `{}` 필요없고, 불필요한 삼항 연산자는 지양하는게 좋아보입니다. 혹시 null을 반환하는 다른 이유가 있으신가요?

Reply:
이전에 멘토님께 코드리뷰 받을 때 && 보다는 삼항연산자를 두어서 조건부 렌더링 하는 것이 안전하다고 하셔서 이렇게 구현하였습니다! alt는 {} 지워 놓을게요 감사합니다!